lldb/test/Shell/SwiftREPL/Inputs/Test.swift

```diff
@@ -0,0 +1,11 @@
+public struct Bar {
+    public init(Baz: Int) { self.Baz = Baz }
```

Comment:
not worth changing, but fwiw Swift uses camelCase for parameter/property/variable names.